theHarvester/theHarvester.py

```diff
@@ -0,0 +1,25 @@
+import asyncio
```

Comment:
same here, this is just relocating the top-level `theHarvester.py` (same reasoning as above)